tests/testDetector.py

```diff
@@ -217,6 +217,36 @@ def testGetCenter(self):
                 if cameraSys == cameraGeom.PIXELS:
                     self.assertAlmostEquals(ctrPixPoint[i], ctrPoint[i])
 
+    def testCopyDetector(self):
+        """Test copyDetector() method
+        """
+        #
+        # Make a copy without any modifications
+        #
+        detector  = DetectorWrapper().detector
+        ndetector = cameraGeom.copyDetector(detector)
+
+        assert detector.getName() == ndetector.getName()
```

Comment:
Sorry, I had thought that pytest handled assert specially;  I'll go back to the unittest conventions.